src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml

```diff
@@ -3,5 +3,4 @@
     ViewBag.Title = "Admin Dashboard";
 }
 
-<h2>Admin Dashboard</h2>
```

Comment:
How does this change relate to the task at hand?

Reply:
No, this cahnge was made before solving the issue

Reply:
So it should not be part of this PR